Automattic/wp-calypso
# VideoPress User intent: some affordances for keyboard navigation

Related to # https://github.com/Automattic/greenhouse/issues/1836

## Proposed Changes

* add hover state on keyboard focus of Intent items
* allow `esc` key to be used to dismiss the modal
* add label to the modal Close button

This may not be exhaustive, but it makes it a little better when using keyboard navigation. Given the short lifespan of this screen it may not be entirely necessary? But I wanted it to be a little better if possible.

## Testing Instructions



* `yarn && yarn start`
* `/setup/videopress`
* use `tab` to move around the intent items, they should highlight (new)
* `enter` or `space` (existing functionality) should open the modal
* `esc` should close the modal (new)

## Pre-merge Checklist
